portal/models/encounter.py

```diff
@@ -172,6 +172,10 @@ def initiate_encounter(user, auth_method):
     if user.has_role(ROLE.SERVICE.value):
         auth_method = 'service_token_authenticated'
 
+    # If the auth_method is unknown, fall back to failsafe
+    if auth_method is None:
+        auth_method = 'failsafe'
```

Comment:
Should this be something like "unknown"?

Reply:
it's part of an enumeration - and in the db - so that would require a migration and other work.  i don't feel it's that semantically different - not worth the effort.  prefer to stick w/ fail safe, which we use when the situation is "unknown".